Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Common properties -->
-  <PropertyGroup>
+  <PropertyGroup Condition=" '$(MRS_DevOptOut)' == '' OR '$(MRS_DevOptOut)' == 'false' ">
```

Comment:
Could you clarify what this flag is for? Is it because msb4u wants to set its own path variables and not get clobbered here?

If so, should we move these into a props file and explicitly include them from each csproj. It'll be a little more verbose, but maybe being explicit is worth it?

Reply:
Exactly, and I certainly am for that option, I just didn't want to start making drastic changes this PR. So this flag allows me to have a closer to source Directory.Build.props and opt out of this one.

Reply:
Filed #116